src/Arcus.Templates.AzureFunctions.Databricks.JobMetrics/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM mcr.microsoft.com/azure-functions/dotnet:4 AS base
 WORKDIR /app
 EXPOSE 80
 
-FROM mcr.microsoft.com/dotnet/sdk:7.0.203-bullseye-slim AS build
+FROM mcr.microsoft.com/dotnet/sdk:7.0.401-bullseye-slim AS build
```

Comment:
I think it's strange that we use a .NET 7 base-layer here.  I thought we'd be using .NET 6 (LTS) and in the very near future .NET 8 (LTS)
Same is true for the other docker files.